CANscope/CZUL/Approches/APP YQB11.asr

```diff
@@ -120,16 +164,21 @@ Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 1 right base
 Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 1 right ticks
 Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 1 right vectoring
 Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 2
+Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 2 left base
 Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 2 left ticks
+Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 2 left vectoring
+Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 2 right base
 Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 2 right ticks
+Runways:CYQB Quebec Jean Lesage 11-29:extended centerline 2 right vectoring
 Runways:CYQB Quebec Jean Lesage 11-29:name
 Sids:YQB arr KAROT:
 Sids:YQB arr OLAVO:
 Sids:YQB arr OMVAR:
 Sids:YQB arr PESAC:
+Sids:YQB dep SEBIP:
```

Comment:
for rwy 06 only

Reply:
https://github.com/rt-2/CZUL-Scope-Files/pull/50/commits/2ba27fe878bb38713bb73a95b9a7f4352ef7222f